resources/authgear/templates/en/web/authflowv2/__phone_input.html

```diff
@@ -43,70 +43,75 @@
+                data-controller="dismiss-keyboard-on-scroll"
+                data-custom-select-target="search"
+                data-action="input->custom-select#search"
+              >
+              <div class="select__search-icon">
+                <i class="material-icons">search</i>
+              </div>
+              <button
+                class="select__clear-icon"
+                type="button"
+                tabindex="-1"
+                data-custom-select-target="clearSearch"
+                data-action="click->custom-select#clearSearch"
+              >
+                <i class="material-icons material-icons--filled">cancel</i>
+              </button>
             </div>
             <button
-              class="select__clear-icon"
+              class="select__close-icon"
               type="button"
               tabindex="-1"
-              data-custom-select-target="clearSearch"
-              data-action="click->custom-select#clearSearch"
+              data-action="click->custom-select#close"
             >
-              <i class="material-icons material-icons--filled">cancel</i>
+              <i class="material-icons">close</i>
             </button>
           </div>
-          <button
-            class="select__close-icon"
-            type="button"
+        </template>
+
+        <template data-custom-select-target="itemTemplate">
+          <li
+            class="select__option"
+            role="option"
+            data-action="
+              click->custom-select#handleSelect
+              keydown->custom-select#handleKeyDown
+            "
             tabindex="-1"
-            data-action="click->custom-select#close"
           >
-            <i class="material-icons">close</i>
-          </button>
-        </div>
-      </template>
-
-      <template data-custom-select-target="itemTemplate">
-        <li
-          class="select__option"
-          role="option"
-          data-action="
-            click->custom-select#handleSelect
-            keydown->custom-select#handleKeyDown
-          "
-          tabindex="-1"
-        >
-          <span data-label="prefix" class="select__option-prefix"></span>
-          <span data-label="content" class="select__option-content"></span>
-        </li>
-      </template>
+            <span data-label="prefix" class="select__option-prefix"></span>
+            <span data-label="content" class="select__option-content"></span>
+          </li>
+        </template>
 
-      <template data-custom-select-target="emptyTemplate">
-        <li class="select__empty">
-          {{ template "v2.component.phone-input.default.no-results-found" }}
-        </li>
-      </template>
+        <template data-custom-select-target="emptyTemplate">
+          <li class="select__empty">
+            {{ template "v2.component.phone-input.default.no-results-found" }}
+          </li>
+        </template>
+      </div>
```

Comment:
Here I just wrap the original dropdown with a `div`. You can hide whitespace change to view the real change